docs/snippets/ov_network_state_intro.py

```diff
@@ -5,19 +5,18 @@
 import numpy as np
 import sys
 
-from openvino.runtime import opset10 as ops
-from openvino.runtime import Core, Model, PartialShape, Tensor, Type
+import openvino as ov
 from openvino.runtime.passes import LowLatency2, MakeStateful, Manager
 
 
 def state_network_example():
     #! [ov:state_network]
-    input = ops.parameter([1, 1], dtype=np.float32)
-    read = ops.read_value(input, "variable0")
-    add = ops.add(read, input)
-    save = ops.assign(add, "variable0")
-    result = ops.result(add)
-    model = Model(results=[result], sinks=[save], parameters=[input])
+    input = ov.opset10.parameter([1, 1], dtype=np.float32)
+    read = ov.opset10.read_value(input, "variable0")
+    add = ov.opset10.add(read, input)
+    save = ov.opset10.assign(add, "variable0")
+    result = ov.opset10.result(add)
+    model = ov.Model(results=[result], sinks=[save], parameters=[input])
```

Comment:
Same story as for opset8, needs to be imported on top or use `ov.runtime.opset10...`